google-cloud-bigtable/acceptance/bigtable/table_test.rb

```diff
@@ -18,62 +18,53 @@
 require "bigtable_helper"
 
 describe "Instance Tables", :bigtable do
-  let(:instance) { bigtable_instance }
+  let(:instance_id) { bigtable_instance_id }
 
   it "create, list all, get table and delete table" do
     table_id = "test-table-#{random_str}"
 
-    table = instance.create_table(table_id) do |cfs|
+    table = bigtable.create_table(instance_id, table_id) do |cfs|
       cfs.add("cf", Google::Cloud::Bigtable::GcRule.max_versions(3))
     end
 
     table.must_be_kind_of Google::Cloud::Bigtable::Table
 
-    tables = instance.tables.to_a
+    tables = bigtable.tables(instance_id).to_a
     tables.wont_be :empty?
     tables.each do |t|
       t.must_be_kind_of Google::Cloud::Bigtable::Table
     end
 
-    table = instance.table(table_id, perform_lookup: true)
+    table = bigtable.table(instance_id, table_id, perform_lookup: true)
     table.must_be_kind_of Google::Cloud::Bigtable::Table
 
     table.delete
-    table = instance.table(table_id)
     table.exists?.must_equal false
   end
 
-  it "create table with initial splits" do
+  it "create table with initial splits and granularity" do
     table_id = "test-table-#{random_str}"
 
     initial_splits = ["customer_1", "customer_10000"]
 
-    table = instance.create_table(table_id, initial_splits: initial_splits) do |cfs|
-      cfs.add("cf", Google::Cloud::Bigtable::GcRule.max_versions(1))
-    end
-
-    table.must_be_kind_of Google::Cloud::Bigtable::Table
-    instance.table(table_id, view: :FULL, perform_lookup: true).wont_be :nil?
-    table.delete
-  end
-
-  it "create table with time granularity" do
-    table_id = "test-table-#{random_str}"
-
-    table = instance.create_table(table_id, granularity: :MILLIS) do |cfs|
+    table = bigtable.create_table(
+        instance_id,
+        table_id,
+        initial_splits: initial_splits,
+        granularity: :MILLIS) do |cfs|
       cfs.add("cf", Google::Cloud::Bigtable::GcRule.max_versions(1))
     end
 
     table.must_be_kind_of Google::Cloud::Bigtable::Table
     table.granularity_millis?.must_equal true
-    instance.table(table_id, view: :NAME_ONLY, perform_lookup: true).wont_be :nil?
+    table.exists?.must_equal true
```

Comment:
It would be ideal to check sample row keys to ensure that they splits took effect.

Reply:
ok I will make change.